js/__internal/grids/grid_core/views/m_grid_view.ts

```diff
@@ -71,19 +82,19 @@ const resizingControllerMembers = {
   },
 
   _refreshSizes(e) {
-    let resizeDeferred;
-    const that = this;
-    const changeType = e && e.changeType;
-    const isDelayed = e && e.isDelayed;
-    const items = that._dataController.items();
+    // @ts-expect-error
+    let resizeDeferred = new Deferred().resolve();
```

Comment:
Let's add types and resolve value here:

```suggestion
    let resizeDeferred = new Deferred<null>().resolve(null);
```